src/main/java/core/creatorclasses/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.creatorclasses;
+
+import core.figureclasses.Circle;
+import core.figureclasses.Figure;
+import core.figureclasses.IsoscelesTrapezoid;
+import core.figureclasses.Rectangle;
+import core.figureclasses.RightTriangle;
+import core.figureclasses.Square;
+import core.interfaces.Constants;
+import java.util.Random;
+
+public class FigureSupplier implements Constants {
+    public Figure getRandomFigure() {
+        Random random = new Random();
+        String figure = FIGURE[random.nextInt(FIGURE_NUMBER)];
```

Comment:
```suggestion
        String figureType = FIGURE[random.nextInt(FIGURE_NUMBER)];
```
так як тип не figure я б тут трохи змінив неймінг